plugins/github/dune

```diff
@@ -11,6 +11,7 @@
    current.term
    current_git
    current_web
+   github-unix
```

Comment:
Using the `ocaml-github` bindings to tidy up the request logic.
The GraphQL API supports the same functionality so we could use that in future.